templates/docs/patterns/grid.md

```diff
@@ -80,3 +105,101 @@ For more information see [Customising Vanilla](/docs/customising-vanilla/) in yo
 ### Design
 
 For more information [view the grid design spec](https://github.com/ubuntudesign/vanilla-design/tree/master/Grid), which includes the specification in markdown format and a PNG image.
+
+## Breaking out of the grid
```

Comment:
As mentioned separately, would be good to split it into separate page.

Reply:
But keep a section on Grid page that links there.

Reply:
done